radio/src/audio_arm.cpp

```diff
@@ -1235,6 +1266,23 @@ void audioEvent(unsigned int index)
   }
 }
 
+#if defined(SDCARD)
+void pushUnit(uint8_t unit, uint8_t idx, uint8_t id=0)
+{
+  if (unit < DIM(unitsFilenames) && idx < 5)
+  {
+    char path[AUDIO_FILENAME_MAXLEN+1];
+    getSystemAudioPath(path);
+    getStringWithIndex(path, unitsFilenames[unit], (int)idx);
+    strcat(path, SOUNDS_EXT);  
```

Comment:
use char * tmp = getStringWithIndex(...)
so that you can call strcpy instead of strcat